backend/conduit/organizations/models.py

```diff
@@ -51,27 +51,18 @@ def add_moderator(self, profile):
             return True
         return False
 
-    def is_moderator(self, profile):
-        return bool(self.moderators.filter(
-            moderator_assoc.c.organization == profile.id).count()
-            )
-
     # Method to add member to the organization
```

Comment:
Please pull from the staging branch so I don't have to merge duplicated code :' ).